runner.go

```diff
@@ -437,7 +437,7 @@ func (r *runner) makeRunParams(c cliContext, localTests []*rainforest.RFTest, br
 		}
 
 		var environment *rainforest.Environment
-		environment, err = r.client.CreateTemporaryEnvironment(description, customURL.String(), webhookURL.String())
+		environment, err = r.client.CreateTemporaryEnvironment(description, customURLParam, webhookURL.String())
```

Comment:
Should we also use `webhookParam` here, since that could be susceptible to the same issue?